### PR TITLE
patch `requires_grad` error for ints

### PR DESCRIPTION
**Pull request by gszep**
this patch bypasses the error https://github.com/pytorch/pytorch/issues/83913. In some cases we cannot assume that `requires_grad=False` in the modified block

### CODE REVIEW

**Comment by TimDettmers** [comment]
The Int4 layers do not have weight gradients. So I am hesitant to merge this since it could introduce silent unexpected behavior in many cases. Maybe I am missing the point here: why do you want to set `requires_grads=True`?

**Comment by gszep** [comment]
Sorry I was being a dum dum. You're right. Closing the PR 😁
